DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.61.136:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.61.136:80 "POST /apitest/signup/ HTTP/1.1" 200 103
DEBUG    root:requests.py:22 发起请求的参数为：{'url': 'http://192.168.61.136:80/apitest/signup/', 'method': 'post', 'json': {'username': 'test08', 'password': '123456', 'confirm': '', 'name': '注册确认密码为空用户'}}
DEBUG    root:asserts.py:17 Response实际结果为：{'Status': 1003, 'Result': 'Confirm password is null', 'Message': '注册失败，确认密码不能为空'}
DEBUG    root:asserts.py:18 Response预期结果为：{'Status': 1003, 'Result': 'Confirm password is null', 'Message': '注册失败，确认密码不能为空'}
INFO     root:asserts.py:20 断言用例通过！
DEBUG    root:mysql.py:146 验库的预期结果：(1, 1)
DEBUG    root:mysql.py:147 验库的实际结果：(1, 1)
INFO     root:mysql.py:149 验证数据库用例通过！